### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Omegaeye**

### CODE REVIEW

File: test/items_test.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      :unit_price => 25,
+      :merchant_id => 0101,
+      :created_at => "AZ",
+      :updated_at => "CO"})
+
+      assert_instance_of Item, @data
+  end
+
+  def test_it_exists
+    @data
```
Comment (b-enji-cmd) [inline test/items_test.rb]: I don't believe we need this lone `@data` 
Otherwise, it looks good!

**Review by b-enji-cmd — APPROVED** [review]
For these small files it won't come into play really but for our bigger repo files. We should get into the habit of having a commit for every method or so.

File: test/items_test.rb (hunk 1 above)
Comment (Omegaeye) [reply test/items_test.rb]: ok we can change that during the next pull request =)

Comment (b-enji-cmd) [reply test/items_test.rb]: > ok we can change that during the next pull request =)

Sounds like a plan!

File: lib/items.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  def initialize (data)
+    @id          = data[:id]
+    @name        = data[:name]
+    @description = data[:description]
+    @unit_price  = data[:unit_price]
+    @merchant_id = data[:merchant_id]
+    @created_at  = data[:created_at]
+    @updated_at  = data[:updated_at]
+  end
+end
```
Comment (tsnieuwen) [inline lib/items.rb:27]: Looks good Khoa - Isn't there also a `unit_price_to_dollars` method in this class?

Comment (Omegaeye) [reply lib/items.rb:27]: yeah, but i think that will go in the items repo file

Comment (tsnieuwen) [reply lib/items.rb:27]: You sure it's not in the Item class? Looking over the iteration 0 guidance it reads like it is. I guess we can always move things around later.

Comment (b-enji-cmd) [reply lib/items.rb:27]: > You sure it's not in the Item class? Looking over the iteration 0 guidance it reads like it is. I guess we can always move things around later.

I think you are correct in that the `unit_price_to_dollars` lies in the `item.rb` file. I can hop on a call if need be! 
image

Comment (carynkeffer) [reply lib/items.rb:27]: I see where Tommy is coming from... there's a method following the list of Item instances with an extra method called unit_price_to_dollars

File: test/items_test.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+require 'CSV'
```
Comment (tsnieuwen) [inline test/items_test.rb]: Do we need to require CSV here? If we're creating our own hash for the test?

Comment (b-enji-cmd) [reply test/items_test.rb]: I think you're right, when we were making the test initially I assumed we might need it so I told him to throw it in, looks like we don't!

**Review by carynkeffer — CHANGES_REQUESTED** [review]
Request `unit_price_to_dollars` method added.

File: test/items_test.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+
+class ItemTest < Minitest::Test
+
+  def setup
+    @data = Item.new({
+      :id => 1,
+      :name => "Pencil",
+      :description => "You can use it to write things",
+      :unit_price => BigDecimal.new(10.99,4),
```
Comment (b-enji-cmd) [inline test/items_test.rb:15]: There is a lot of tricky formatting for Items,  not quite sure how to use BigDecimal properly.

**Review by tsnieuwen — APPROVED** [review]
Looks good to me

File: lib/items.rb [hunk 5]
```diff
@@ -0,0 +1,27 @@
+require 'time'
+require 'bigdecimal/util'
```
Comment (carynkeffer) [inline lib/items.rb:2]: Looking forward to finding out what this means!

**Review by carynkeffer — APPROVED** [review]

**Pull request merged**